smartsim/_core/_cli/cli.py

```diff
@@ -95,16 +96,22 @@ def default_cli() -> SmartCli:
         MenuItemConfig(
             "dbcli",
             "Print the path to the redis-cli binary",
-            dbcli_execute
+            dbcli_execute,
         ),
         MenuItemConfig(
             "site",
             "Print the installation site of SmartSim",
-            site_execute),
+            site_execute,
+        ),
         MenuItemConfig(
             "clobber",
             "Remove all previous dependency installations",
-            clobber_execute
+            clobber_execute,
+        ),
+        MenuItemConfig(
+            "test",
```

Comment:
This is a great idea! Nevertheless, I wouldn't want users to believe this is our official way of running tests. Can we change it's name to something more restrictive, such as `smoke-test`, `dry-run`, `mini-test`, or similar?

Reply:
I thought there was a proposal to change it to `test` to mimic the gmake pattern of `make install; make test`. More than willing to revert to `smart validate` though!